Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -120,8 +122,8 @@ public static Client ToClientViewModel(this IS4.Client client)
                 PostLogoutRedirectUris = client.PostLogoutRedirectUris,
                 EnableLocalLogin = client.EnableLocalLogin,
                 IdentityProviderRestrictions = client.IdentityProviderRestrictions,
-                LogoutUri = client.LogoutUri,
-                LogoutSessionRequired = client.LogoutSessionRequired,
+                LogoutUri = client.FrontChannelLogoutUri ?? client.BackChannelLogoutUri,

```

Comment:
So we aren't using the hybrid, we are using either implicit flow or authorization code flow?

Reply:
Same as the comment above, we weren't sure which was used by our system. It could be that BackChannel is never used, we just were updating to try and preserve the current functionality as closely as possible (and before there was only one URL). I would be fine removing the null coalesce if one is never used by us.